src/ResourceManager/Profile/Commands.Profile.Test/ArgumentCompleterTests.cs

```diff
@@ -12,48 +12,43 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
-using Microsoft.Azure.Commands.ResourceManager.Common;
-using Microsoft.Azure.Commands.Resources.Test.ScenarioTests;
-using Microsoft.Azure.Commands.ScenarioTest;
-using Microsoft.Azure.ServiceManagemenet.Common.Models;
+using  Microsoft.Azure.Commands.TestFx;
 using Microsoft.WindowsAzure.Commands.ScenarioTest;
-using Microsoft.WindowsAzure.Commands.Test.Utilities.Common;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace Microsoft.Azure.Commands.Profile.Test
 {
-    public class ArgumentCompleterTests : RMTestBase
+    public class ArgumentCompleterTests
     {
-        private XunitTracingInterceptor xunitLogger;
+        private readonly ITestRunnable _testManager;
```

Comment:
_testRunner.  Let's use a common pattern in all the places we are using the new faclity, wither:

(1) Have all the test classes extend TestManager or
(2) Have all the test classes create an ITestRunner in their constructor.

This will make things simpler for our developers - they will have only one pattern to follow.